pages/index.js

```diff
@@ -1,7 +1,9 @@
+import Dashboard from "@/components/Dashboard";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
```

Comment:
I guess you don't need the Fractions here.
You just need them, when there are multiple things inside the return